src/issue/issue.action.js

```diff
@@ -172,6 +173,28 @@ export const editIssueComment = (issueCommentId, owner, repoName, body) => {
   };
 };
 
+export const editIssueBody = (owner, repoName, issueNum, body) => {
```

Comment:
`editIssue` currently already exists and I'm definitely not a fan of how I'm passing `updateParams` to it. Because the comment is HTML, can't leverage that to pass in a new update parameter for the text to replace state. 

Will need to refactor how `editIssue` works later without a doubt and instead of passing in updateParams, just get what's returned from the API (and we can then just remove this)